agave_app/GLView3D.cpp

```diff
@@ -164,9 +164,9 @@ GLView3D::paintGL()
     return;
   }
 
-  makeCurrent();
+  // makeCurrent();
   m_viewerWindow->redraw();
-  doneCurrent();
+  // doneCurrent();

```

Comment:
Are these commented out for testing or is this part of the code change?

Reply:
It's part of the code change. I'll delete them.